test/test-pipe-bind-error.c

```diff
@@ -164,10 +164,12 @@ static void connect_overlong_cb(uv_connect_t* connect_req, int status) {
 
 
 TEST_IMPL(pipe_overlong_path) {
-  char path[512];
   uv_pipe_t pipe;
   uv_connect_t req;
 
+#ifndef _WIN32
+  char path[512];
+
   memset(path, '@', sizeof(path));
   ASSERT_OK(uv_pipe_init(uv_default_loop(), &pipe, 0));
```

Comment:
```suggestion
  ASSERT_OK(uv_pipe_init(uv_default_loop(), &pipe, 0));

#ifndef _WIN32
  char path[512];
  memset(path, '@', sizeof(path));
```
Should work because tests are compiled in C99 mode, I believe? Anyway, the test failures are due to missing uv_pipe_init() call.